### PR TITLE
Increase number of excludable paths from 3 to 12

### PR DESCRIPTION
**Pull request by eljenso**
Three excludable paths are too few for my personal use (gf uses same Kodi, but I don't want her shows to be synced), so I increased the number to 12.

Tested and seems to work.

### CODE REVIEW

File: utilities.py [hunk 1]
```diff
@@ -116,31 +116,77 @@ def checkExclusion(fullpath):
...
             logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 2.")
             return True
-
     ExcludePath3 = getSetting('ExcludePath3')
     if ExcludePath3 != "" and getSettingAsBool('ExcludePathOption3'):
         if fullpath.find(ExcludePath3) > -1:
             logger.debug("checkExclusion(): Video is from location, which is currently set as excluded path 3.")
             return True
+    ExcludePath4 = getSetting('ExcludePath4')
+    if ExcludePath4 != "" and getSettingAsBool('ExcludePathOption'):
```
Comment (anaconda) [inline utilities.py]: `ExcludePathOption4` - same for all others below.

**Comment by anaconda** [comment]
@Razzeee I think this `.find` logic is wrong and you actually wanted to use `.startswith`:
```python
>>> excluded = '/hey/'
>>> path = '/storage/test/hey/1x01.mkv'
>>> path.find(excluded) > -1
True
>>> path.startswith(excluded)
False
```

File: utilities.py (hunk 1 above)
Comment (Razzeee) [inline utilities.py]: Your right

**Comment by Razzeee** [comment]
I think I never touched this part since I took over the project. And yeah, startswith should be better.

**Pull request merged**